src/lib/middleware/api-token-middleware.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
The purpose of this middleware is to convert `api-tokens` from the `authorization` header and convert that in to a api-user if it is a valid token. The api-user might or might not have access to the requested resource, but that is out of scope for this feature.